src/MetaModels/Helper/TableManipulation.php

```diff
@@ -74,6 +74,21 @@ class TableManipulation
     const STATEMENT_DROP_COLUMN = 'ALTER TABLE %s DROP COLUMN %s';
 
     /**
+     * SQL statement template to add a index to a column of a table.
+     * First parameter is table name.
+     * second parameter is indextype
+     * third parameter is name of the column.
+     */
+    const STATEMENT_ADD_INDEX_COLUMN = 'ALTER TABLE %s ADD %s(%s)';
+    
+    /**
+     * SQL statement template to drop a fulltext index from a column of a table.
+     * First parameter is table name.
+     * Second parameter is the name of the column, witch is used as index name
+     */
+    const STATEMENT_DROP_INDEX_COLUMN = 'ALTER TABLE %s DROP INDEX %s';
```

Comment:
Did you forget to implement the method `dropIndex()` by accident or why did you add this constant?